task/power/src/bsp/cosmo_a.rs

```diff
@@ -0,0 +1,343 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Based on the `gimlet_bcdef.rs` implementation in this folder
+use crate::{
+    i2c_config, i2c_config::sensors, Device, PowerControllerConfig, PowerState,
+    SensorId,
+};
+
+use drv_i2c_devices::max5970::*;
+use ringbuf::*;
+use userlib::units::*;
+
+pub(crate) const CONTROLLER_CONFIG_LEN: usize = 42;
+const MAX5970_CONFIG_LEN: usize = 22;
+
+pub(crate) static CONTROLLER_CONFIG: [PowerControllerConfig;
+    CONTROLLER_CONFIG_LEN] = [
+    rail_controller!(IBC, bmr491, v12_sys_a2, A2),
+    rail_controller!(Core, raa229620A, vddcr_cpu0_a0, A0),
+    rail_controller!(Core, raa229620A, vddcr_soc_a0, A0),
+    rail_controller!(Core, raa229620A, vddcr_cpu1_a0, A0),
+    rail_controller!(Core, raa229620A, vddio_sp5_a0, A0),
+    rail_controller_notemp!(Core, isl68224, v1p1_sp5_a0, A0),
+    rail_controller_notemp!(Core, isl68224, v1p8_sp5_a1, A0), // XXX A0 or A2?
+    rail_controller_notemp!(Core, isl68224, v3p3_sp5_a1, A0), // XXX A0 or A2?
+    rail_controller!(Sys, tps546B24A, v3p3_sp_a2, A2),
+    rail_controller!(Sys, tps546B24A, v5_sys_a2, A2),
+    rail_controller!(Sys, tps546B24A, v1p8_sys_a2, A2),
+    rail_controller!(Sys, tps546B24A, v0p96_nic_vdd_a0hp, A0),
+    adm1272_controller!(HotSwap, v54p5_ibc_a3, A2, Ohms(0.000075)),
```

Comment:
My first comment would be that it's worth putting an underscore in there -- e.g., `Ohms(0.000_075)` -- but my second comment would be that I *think* that the value is incorrect? R738 is 750µΩ -- i.e., `Ohms(0.000_750)`

Reply:
Good catch, fixed!